frontend/apps/erd-web/app/.well-known/vercel/flags/route.ts

```diff
@@ -0,0 +1,16 @@
+import { type ApiData, verifyAccess } from 'flags'
+import { getProviderData } from 'flags/next'
+import { type NextRequest, NextResponse } from 'next/server'
+// biome-ignore lint/nursery/useImportRestrictions: <explanation>
+import * as flags from '../../../../libs/flags'
```

Comment:
Wasn't it possible to write it as follows?

```suggestion
import * as flags from '@/libs/flags'
```

Reply:
You're right! I'll fix it.

Reply:
fixed: https://github.com/liam-hq/liam/pull/830/commits/1ff239e47b9069fccaff67e2674dba02e49b3402